src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  padding-bottom: 50px;
+  padding-top: 50px;
+
+  &__box {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    width: $catalog-width;
+  }
```

Comment:
Why not to set this styles to catalog (except margin, you may use BEM element for it), I don't think that you really need this container